Sources/TokamakCore/Views/List.swift

```diff
@@ -0,0 +1,261 @@
+
+//  public init<Data, RowContent>(_ data: Data,
+//                                children: KeyPath<Data.Element, Data?>,
+//                                selection: Binding<SelectionValue?>?,
+//                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+//  where Content == OutlineGroup<Data,
+//                                Data.Element.ID,
+//                                HStack<RowContent>,
+//                                HStack<RowContent>,
+//                                DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+//        Data : RandomAccessCollection, RowContent : View, Data.Element : Identifiable
+
+//  public init<Data, ID, RowContent>(_ data: Data,
+//                                    id: KeyPath<Data.Element, ID>,
+//                                    children: KeyPath<Data.Element, Data?>,
+//                                    selection: Binding<SelectionValue?>?,
+//                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+//  where Content == OutlineGroup<Data,
+//                                ID,
+//                                HStack<RowContent>,
+//                                HStack<RowContent>,
+//                                DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+//        Data : RandomAccessCollection, ID : Hashable, RowContent : View
+}
+
+extension List where SelectionValue == Never {
+  public init(@ViewBuilder content: () -> Content) {
+    selection = .one(nil)
+    self.content = content()
+  }
+
+  public init<Data, RowContent>(_ data: Data,
+                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == ForEach<Data, Data.Element.ID, HStack<RowContent>>,
+    Data: RandomAccessCollection, RowContent: View, Data.Element: Identifiable {
+    selection = .one(nil)
+    content = ForEach(data) { row in
+      HStack {
+        rowContent(row)
+      }
+    }
+  }
+
+  // FIXME: Implement OutlineGroup
+//  public init<Data, RowContent>(_ data: Data,
+//                                children: KeyPath<Data.Element, Data?>,
+//                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+//  where Content == OutlineGroup<Data,
+//                                Data.Element.ID,
+//                                HStack<RowContent>,
+//                                HStack<RowContent>,
+//                                DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+//        Data : RandomAccessCollection, RowContent : View, Data.Element : Identifiable
+//  public init<Data, ID, RowContent>(_ data: Data,
+//                                    id: KeyPath<Data.Element, ID>,
+//                                    children: KeyPath<Data.Element, Data?>,
+//                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/List.swift;
  Line: 230;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 102 characters (`line_length`)